modules/app.js

```diff
@@ -1,5 +1,6 @@
 /* eslint-disable no-plusplus */
 /* eslint-disable radix */
```

Comment:
- [ ] It is not best practice to disable some rules just to bypass the linter check. This is because doing so will bring some syntax errors in your code and **you will not be aware of it because eslint is unaware of it.** **As Developers, we should work as we can in order to correct errors marked by linters instead of disabling them** 😉